tests/test_notifications.py

```diff
@@ -833,6 +833,14 @@ def test_notify_does_not_send_comment_if_they_reply_to_their_own_comment(self, m
         mock_send.assert_called_with([self.project.creator._id], 'email_transactional', self.project._id, 'comment_replies', commenter=self.project.creator, target_user=self.project.creator)
         assert_false(mock_send_mail.called)
 
+    @mock.patch('website.notifications.emails.send')
+    def test_notify_sends_comment_event_if_comment_reply_is_not_direct_reply_on_component(self, mock_send):
+        """ Test that comment replies on components that are not direct replies to the subscriber use the
```

Comment:
> Use long, descriptive names. This often obviates the need for docstrings in test methods. This also makes it easier to locate tests that fail.

http://cosdev.readthedocs.org/en/latest/process/testing.html#general-testing-guidelines

Docstrings can be replaced with `#` comments.